### DISCUSSION TITLE
certificate verification failed

### DISCUSSION THREAD

**Discussion by ghost** [ente-io/ente#3714] [Q&A]
hello ente community,

I have just reinstalled ente auth for the third time because it gave an error that my certificate cannot be verified (see screenshot) when I then click on the ‘update certificates’ button I get an error that Microsoft Edge cannot read and write to the associated folder. does anyone know a solution? I have updated the logs for if anyone is interested in them. i am annoyed of having to use the web app all the time.

Kind regards, wilm

full translation error message fromEdge

the data folder cannot be created
Microsoft Edge cannot read from and write to the associated data folder: C:\Program Files\Ente Auth\webview_window_WebView2\EBWebView

[ente-logs-2024-Oct-15-13-10.zip](https://github.com/user-attachments/files/17378544/ente-logs-2024-Oct-15-13-10.zip)
![Schermafbeelding 2024-10-15 150744](https://github.com/user-attachments/assets/94300786-1502-4cd8-8761-5a752a5f4dc8)
![Schermafbeelding 2024-10-15 150759](https://github.com/user-attachments/assets/18def2e8-b1ad-40dd-b148-dbb5d53825e9)

**Comment by ua741** [comment]
Can you follow the steps [here](https://help.ente.io/auth/troubleshooting/windows-login) and confirm if that resolves the issue?

**Reply by ghost** [reply]
thanks, everything works again.😃 i can finally do all my things again without opening the webapp (not that there's anything wrong with that, just that it takes more time)
